src/main/java/io/electrum/vas/model/An32TokenPayment.java

```diff
@@ -32,34 +34,36 @@ public void setToken(String token) {
       this.token = token;
    }
 
-   @Override
-   public int hashCode() {
-      return Objects.hash(name, type, token);
-   }
-
    @Override
    public boolean equals(Object o) {
-      if (this == o) {
+      if (this == o)
          return true;
-      }
-      if (o == null || getClass() != o.getClass()) {
+      if (o == null || getClass() != o.getClass())
          return false;
-      }
-      An32TokenPayment otherToken = (An32TokenPayment) o;
-      return Objects.equals(this.type, otherToken.type) && Objects.equals(this.name, otherToken.name)
-            && Objects.equals(this.token, otherToken.token) && Objects.equals(this.amount, otherToken.amount);
+      if (!super.equals(o))
+         return false;
+      An32TokenPayment that = (An32TokenPayment) o;
+      return Objects.equals(token, that.token);
+   }
+
+   @Override
+   public int hashCode() {
+      return Objects.hash(super.hashCode(), token);
    }
 
    @Override
    public String toString() {
       StringBuilder sb = new StringBuilder();
       sb.append("class An32TokenPayment {\n");
-
-      sb.append("    type: ").append(Utils.toIndentedString(type)).append("\n");
       sb.append("    name: ").append(Utils.toIndentedString(name)).append("\n");
+      sb.append("    type: ").append(Utils.toIndentedString(type)).append("\n");
       sb.append("    amount: ").append(Utils.toIndentedString(amount)).append("\n");
+      sb.append("    issuer: ").append(Utils.toIndentedString(issuer)).append("\n");
+      sb.append("    pin: ").append(Utils.toIndentedString(pin)).append("\n");
+      sb.append("    proxy: ").append(Utils.toIndentedString(proxy)).append("\n");
+      sb.append("    proxyType: ").append(Utils.toIndentedString(proxyType)).append("\n");
       sb.append("    token: ").append(Utils.toIndentedString(token)).append("\n");
-      sb.append("}");
+      sb.append('}');
```

Comment:
ah, you fixed the curly brace but not the newlines

```suggestion
      sb.append("    name: ").append(Utils.toIndentedString(name)).append('\n');
      sb.append("    type: ").append(Utils.toIndentedString(type)).append('\n');
      sb.append("    amount: ").append(Utils.toIndentedString(amount)).append('\n');
      sb.append("    issuer: ").append(Utils.toIndentedString(issuer)).append('\n');
      sb.append("    pin: ").append(Utils.toIndentedString(pin)).append('\n');
      sb.append("    proxy: ").append(Utils.toIndentedString(proxy)).append('\n');
      sb.append("    proxyType: ").append(Utils.toIndentedString(proxyType)).append('\n');
      sb.append("    token: ").append(Utils.toIndentedString(token)).append('\n');
      sb.append('}');
```